web/src/ee/features/evals/server/addDatasetRunItemsToEvalQueue.ts

```diff
@@ -18,30 +21,17 @@ export const addDatasetRunItemsToEvalQueue = async ({
     const queue = DatasetRunItemUpsertQueue.getInstance();
 
     if (queue) {
-      await queue.add(
-        QueueJobs.DatasetRunItemUpsert,
-        {
-          payload: {
-            projectId,
-            datasetItemId: datasetItemId,
-            traceId,
-            observationId: observationId ?? undefined,
-          },
-          id: randomUUID(),
-          timestamp: new Date(),
-          name: QueueJobs.DatasetRunItemUpsert as const,
+      await queue.add(QueueJobs.DatasetRunItemUpsert, {
+        payload: {
+          projectId,
+          datasetItemId: datasetItemId,
+          traceId,
+          observationId: observationId ?? undefined,
         },
-        {
-          attempts: 5, // retry 3 times
-          backoff: {
-            type: "exponential",
-            delay: 1000,
-          },
-          delay: 30000, // 10 seconds
-          removeOnComplete: true,
-          removeOnFail: 1_000,
-        },
```

Comment:
We fill fallback to the defaults on the queue. No need to overwrite for each item.